File: cadwiki-nuget/cadwiki.NUnitTestRunner/HTML_Models/TestSuiteReport.vb
Namespace HTML_Models
    Public Class TestSuiteReport
        Public Title As String
        Public BannerImagePath As String
        Public TabName As String
        Public PassPercentage As String
        Public DateString As String
        Public TestSuiteResults As Results.ObservableTestSuiteResults

        Public HeaderBackgroundColor As String = ""


        Public Sub New(results As Results.ObservableTestSuiteResults)
            Title = results.TestSuiteName
            TabName = Title
            PassPercentage = ((results.PassedTests / results.TotalTests * 1.0) * 100).ToString("0.00") + "%"
            DateString = DateTimeOffset.Now.ToString()
            TestSuiteResults = results
        End Sub

    End Class

    Public Class HexColors
        Public DarkBlue As String = "#002554"
        Public White As String = "#ffffff"
        Public LightBlue As String = "#0093db"
    End Class

End Namespace

File: cadwiki-nuget/cadwiki.NUnitTestRunner/HTML_Models/TestSuiteReport.cs
using System;

namespace cadwiki.NUnitTestRunner.HTML_Models
{
    public class TestSuiteReport
    {
        public string Title;
        public string BannerImagePath;
        public string TabName;
        public string PassPercentage;
        public string DateString;
        public Results.ObservableTestSuiteResults TestSuiteResults;

        public string HeaderBackgroundColor = "";


        public TestSuiteReport(Results.ObservableTestSuiteResults results)
        {
            Title = results.TestSuiteName;
            TabName = Title;
            PassPercentage = (results.PassedTests / (double)results.TotalTests * 1.0d * 100d).ToString("0.00") + "%";
            DateString = DateTimeOffset.Now.ToString();
            TestSuiteResults = results;
        }

    }

    public class HexColors
    {
        public string DarkBlue = "#002554";
        public string White = "#ffffff";
        public string LightBlue = "#0093db";
    }

}